docs/src/calling-contracts/read-only.md

```diff
@@ -0,0 +1,15 @@
+# Read-only calls
```

Comment:
Not sure that `Read only` is the proper way to call this. Can be decided later though.

Reply:
I think so too, couldn't think of anything better for now so I kept the current naming.